windows/src/developer/TIKE/dialogs/UfrmOptions.pas

```diff
@@ -290,6 +244,9 @@ procedure TfrmOptions.cmdOKClick(Sender: TObject);
     UseTabChar := chkUseTab.Checked;
     IndentSize := StrToIntDef(editIndentSize.Text, 4);
     LinkFontSizes := chkLinkFontSizes.Checked;
+    if cbEditorTheme.ItemIndex = cbEditorTheme.Items.Count - 1
+      then EditorTheme := lblEditorCustomTheme.Caption
+      else EditorTheme := SDefaultEditorThemes[cbEditorTheme.ItemIndex].Name;
```

Comment:
indentation?

Reply:
It's a funny indentation style, but one we've used for a while when we have an if/else with both branches being a single line. It's one less line than the traditional layout below and I would argue slightly easier to read!

This is more traditional layout:
```
if ... then
  ...
else
  ...;
```

For multi line we go with:
```
if ... then
begin
  ...;
end
else
begin
  ...;
end;
```